pkg/annotations/annotations.go

```diff
@@ -0,0 +1,66 @@
+package annotations
+
+const (
+	// Annotations carries the received Kubelet annotations
+	Annotations = "io.kubernetes.cri-o.Annotations"
```

Comment:
yes, I intentionally prefixed it with `io.kubernetes` because it's a k8s subproject.